src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -0,0 +1,19 @@
+import { Comment } from '../../types/types';
+
+import './CommentInfo.scss';
+
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => (
+  <>
+    <div className="comment">
+      <p>{comment.name}</p>
+      <p>{comment.body}</p>
+      <p>
+        <a href="..." className="comment__email">{comment.email}</a>
```

Comment:
```suggestion
        <a href={comment.email} className="comment__email">{comment.email}</a>
```
для чого тут три крапки?

Reply:
сделал временную заглушку для ссылки, чтобы линтер не ругался. а исправить забыл (..